scripts/openstack-quickstart-demosetup

```diff
@@ -652,17 +654,29 @@ do
     start_and_enable_service $s
 done
 
+# migrate data directly
+nova-manage db online_data_migrations
+
```

Comment:
Don't you also have to map `cell0` as in [1]
```
nova-manage cell_v2 map_cell0 --database_connection \
  mysql+pymysql://root:secretmysql@dbserver/nova_cell0?charset=utf8
```

[1] https://docs.openstack.org/nova/latest/user/cells.html

Reply:
@nicolasbock I'm still trying my best to translate the corresponding devstack commit: 
https://review.openstack.org/#/c/440739/ but I don't have it owrking yet. and even with that its not avoiding the restart. the restart is the real uglyness imho.